run.py

```diff
@@ -6,4 +6,4 @@
     format = os.getenv('BEHAVE_FORMAT', 'progress2')
 
     behave_executable.main(args=f'--format {format} system_tests/features')
-    behave_executable.main(args=f'--format {format} acceptance_tests/features')
+    behave_executable.main(args=f'--format {format} acceptance_tests/features/notification_letter.feature')
```

Comment:
Is this right?

Reply:
Nope. That was when I was trying to fix the notification feature